server/api/src/main/scala/cool/graph/util/logging/LogData.scala

```diff
@@ -16,11 +16,7 @@ case class LogData(
     requestId: String,
     projectId: Option[String] = None,
     payload: Option[Map[String, Any]] = None
-) {
```

Comment:
Note: For some reason this didn't render the correct JSON anymore, hence the changes.